src/app/components/board/board.component.ts

```diff
@@ -62,14 +64,24 @@ export class BoardComponent implements OnInit {
     return piecePosition;
   }
 
-  paintPiece(x: number, y: number) {
-    this.boardMatrix[x][y] = this.activePlayer.identifier;
-    if (this.isAWinner(x, y)) {
-      this.activePlayer.winner = true;
-      this.isThereAWinner = true;
-    } else {
-      this.setNextTurn();
-    }
+  animateFall(startPosition: Coordenate, endPosition: Coordenate) {
+    setTimeout(() => {
```

Comment:
You have a typo. It's not Coordenate, it's COORDINATE. Please change it in all occurrences